pkg/didcomm/protocol/didexchange/didex_default.go

```diff
@@ -0,0 +1,14 @@
+// +build !interop
```

Comment:
build tag should be more specifically named.

Reply:
Done